.github/workflows/python_tests.yml

```diff
@@ -153,50 +148,3 @@ jobs:
         working-directory: ./sdks/python
         shell: bash
         run: python -m apache_beam.examples.wordcount --input MANIFEST.in --output counts
-
-  python_wordcount_dataflow:
```

Comment:
To clarify, we are removing this since this a duplicate test that is already run elsewhere ?

Reply:
@chamikaramj Thank you for reviewing. Yes. That's correct.